Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_hole.h

```diff
@@ -426,7 +496,7 @@ bool use_dt3 =
 
   /*!
   \ingroup  hole_filling_grp
-  same as above but the range of third points is omitted. They are not
+  Same as above but the range of third points is omitted. They are not
```

Comment:
Check in the html that it is really "above".

Reply:
This is a change that I included from the new version of PMP with the updated named parameters, but I can change it back.